samples/nrf9160/mqtt_simple/README.rst

```diff
@@ -0,0 +1,79 @@
+.. _mqtt_simple_sample:
+
+nRF9160: Simple MQTT
+####################
+
+The Simple MQTT sample demonstrates how to easily connect an nRF9160 SiP to an MQTT broker and send and receive data.
+
+Overview
+*********
+
+The sample connects to an MQTT broker and publishes whatever data it receives on the configured subscribe topic to the configured publish topic.
+
+Requirements
+************
+
+* The following development board:
+
+  * nRF9160 DK board (PCA10090)
+
+* :ref:`secure_boot` must be programmed on the board.
+
+  The sample is configured to compile and run as a non-secure application on nRF91's Cortex-M33.
+  Therefore, it requires the :ref:`secure_boot` bootloader that prepares the required peripherals to be available for the application.
+
+Building and running
+********************
+
+This sample can be found under :file:`samples/nrf9160/mqtt_simple` in the |NCS| folder structure.
+
+The sample is built as a non-secure firmware image, thus with ``CONFIG_TRUSTED_EXECUTION_NONSECURE=y`` set in ``prj.conf``.
```

Comment:
This part is no longer valid and was recently replaced in other samples, please replace this line with the following:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blame/master/samples/nrf9160/asset_tracker/README.rst#L71